docs/django-admin/roles.md

```diff
@@ -9,6 +9,28 @@ our `user_group` model and run in a migration.
 
 For more details, refer to the [user group model](../../src/registrar/models/user_group.py).
 
+## Adding a user as analyst or granting full access via django-admin
+
+If a new team member has joined whose data is NOT included in fixtures,
+then they will need to be granted analyst ( `cisa_analysts_group`) or full access (`full_access_group`) permissions in order to view the django-admin pages.
```

Comment:
**question**: I think that a `User` object also needs to be marked `is_staff` to be able to log in to admin pages. Can we add that to these instructions?

Reply:
Good call out, just DM-ed rachid to confirm this, but if this is the case, it definitely shows why a readme is needed, I didn't realize we still had to add is_staff and assumed that would be covered under permissions.

Reply:
**nit:** TOTALLY an extreme nitpick, but removing the space between ( and `:

( `cisa_analysts_group`) -> (`cisa_analysts_group`)